vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java

```diff
@@ -602,7 +594,7 @@ public MassPieceDefiner() {
    * template.
    *
    */
-  class DefineDialog extends JDialog {
+  static class DefineDialog extends JDialog {
```

Comment:
Should this be private?

Reply:
Yes indeed. We really don't want people subclassing internal parts of the mass Piece Loader. I have made all the internal classes private